config/locales/en.yml

```
@@ -958,13 +958,11 @@ headings.webauthn_setup.new: Insert your security key
 help_text.requested_attributes.address: Address
 help_text.requested_attributes.all_emails: Email addresses on your account
 help_text.requested_attributes.birthdate: Date of birth
+help_text.requested_attributes.consent_reminder_html: You must consent each year to share your information with <strong>%{sp}</strong>.
```

as long as we are here, WDYT about moving the HTML tags out of the translation text? Since it's more likely to come back roundtrip through i18n process and not need manual intervention
```suggestion
help_text.requested_attributes.consent_reminder_html: You must consent each year to share your information with %{sp_html}.
```

and then do 
```ruby
t('....', sp_html: strong_tag(sp.friendly_name))
```

---

> as long as we are here, WDYT about moving the HTML tags out of the translation text? 

Sure 👍  Updated in 50abafed61